index.html

```diff
@@ -0,0 +1,368 @@
+</head>
+<body>
+
+<section id="abstract">
+## Abstract
+
+This explainer discusses new device [=form factors=] that are emerging and related terminology that is being established. This document proposes one naming scheme to be able to reason about and use these terms and concepts consistently in [related web specifications](#related-web-specifications). The use of these terms in other contexts may differ.
+</section>
+
+<!-- How to edit this file: https://respec.org/docs/ -->
+
+## Terminology and conceptual model
+
+><dfn data-export="">Form factor</dfn> is a hardware design aspect that defines and prescribes the size, shape, and other physical specifications of components, particularly in electronics. A form factor may represent a broad class of similarly sized components, or it may prescribe a specific standard. It may also define an entire system, as in a computer form factor. ([Wikipedia](https://en.wikipedia.org/wiki/Form_factor_(design)))
+
+![](images/relationship.svg)
+*<br>Relationship of form factor-related terminology in this explainer.*
+
+### Foldable device
+
+A <dfn data-export="">foldable device</dfn> is a top-level concept that encompasses both a [=dual-screen device=] and a [=foldable-screen device=] [=form factors=].
+
+<img src="images/form-factors.svg" height="150px" alt="">
+
+*<br>A concept of a [=foldable-screen device=] on the left and a concept of a [=dual-screen device=] on the right. ([Source](https://w3c.github.io/screen-fold/))*
+
+A [=foldable device=] has a mechanism that allows the device to fold open and closed similarly to traditional laptops. To differentiate from the traditional laptop form factor, a foldable device has a screen on both sides of the fold. In the most typical configuration, a foldable device can fold a one bigger screen into two smaller screens. A foldable device can have one or more physical internal screens made of solid materials (e.g. glass) or foldable materials (e.g. organic compound). Screens made of solid materials employ a physical hinge between the screens that creates a physical gap between the screens, while the screens made of flexible materials can bend without forming a physical gap between the logical regions divided by the fold. The physical hinges can rotate from ~0 degree to at most 360 degrees, but the range of movement may be limited depending on the device’s hardware characteristics. Screens made of flexible material can also bend within certain limits and to differentiate from the designs with a physical hinge, they do not have a physical gap between the logical regions divided by the fold. The number of logical regions the screen may fold into is not limited and the physical size of the device screen and its aspect ratio is not restricted. This translates into a wide spectrum of possible form factors and designs. Given this diversity, great care should be taken when exposing these capabilities to web developers to ensure a future-proof and extensible design.
```

Comment:
" Screens made of flexible material can also bend within certain limits and to differentiate from the designs with a physical hinge, they do not have a physical gap between the logical regions divided by the fold."

This sentence is confusing.
The wording "to differentiate" doesn't seem to be proper.
single flexible material screen, do also have a hinge, not visible, but they do have.
Therefore I would suggest that:
- All foldable devices have a physical hinge.
- Multi/Dual screens devices have a VISIBLE physical hinge with a gap
- Foldable screen devices have a NON VISIBLE physical hinge with no gap.
Maybe redundant wording here, if we consider that  "visible hinge" = "gap."

Reply:
Good suggestions. I reworded this section, PTAL. I have a feeling it's be nice to have a complementary table with form factors (dual-screen device, foldable-screen device) on one axis and features (visible physical hinge, gap etc.) on the other. Maybe something to add in subsequent updates.